src/mongocxx/write_concern.hpp

```diff
@@ -55,6 +55,15 @@ class bulk_write;
 class MONGOCXX_API write_concern {
 
    public:
+    ///
+    /// A class to represent the special case values for write_concern::nodes.
+    /// @see http://docs.mongodb.org/manual/reference/write-concern/#w-option
+    ///
+    enum class level : std::int32_t {
+        kUnacknowledged = 0,
+        kDefault = -2,
```

Comment:
i think I was wrong about our enum style before. While the server uses e.g. 'kDefault', the driver should use 'k_default',